QUERY_EXTENSION.md

```diff
@@ -0,0 +1,34 @@
+
+Query extension is added in Query directory instead of extensions folderso that changes are additive only
+and can be applied to other extensions with minimal change
+
+## Extending the Schema
+
+Extensions are additional categories, event classes, attributes, objects or profiles. The Open
+Cybersecurity Schema Framework can be extended by adding new attributes, objects, categories
+and event classes. A schema is the aggregation of core schema entities and extensions.
+Extensions allow a particular vendor or customer to create a new schema or augment an existing
+schema. Extensions can also be used to factor out non-essential schema domains keeping a
+schema small. 
+
+Query extension can be use to extend events, objects, attributes of any of the namespace schema.
+namespace is an optional attribute in events, objects, includes which can be added to extend an extension's schema.
```

Comment:
I think the first paragraph should be removed or edited, and we should be clear how/why the OCSF "extensions" aren't sufficient for what we're doing. Otherwise "query extension" seems to be the same thing as "OCSF extension", and readers might wonder why we didn't just make an OCSF extension.

> The Open Cybersecurity Schema Framework can be extended by adding new attributes, objects, categories and event classes

vs

> Query extension can be use to extend events, objects, attributes of any of the namespace schema.